modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,585 @@
+
+You can retrieve the conflict logging settings for a replication using the REST API `GET /settings/replications/`.
+For more information, see xref:rest-api:rest-xdcr-adv-settings.adoc#rest-api-conflict-logging-replication-tunables[Retrieve Conflict Logging Settings].
+The conflict logger settings start with the prefix `cLog`.
+
+NOTE: Couchbase recommends that you must not adjust the conflict logger settings unless advised to do so by the Couchbase Support.
+
+The following table explains the conflict logger settings.
+The `conflictLogging` setting, used to enable and configure logging of conflicts, is explained in xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#configure-conflictlogging-settings[Enabling and Configuring Conflict Logging].
+
+|===
+| Setting | Description | Default
+
+| `conflictLogging`
+| The main setting which when non-empty and valid enables logging of conflicts.
+
+For more information, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflictlogging-key[Replication Setting conflictLogging key].
+|
+
+| `cLogSetTimeoutMs`
+| Sets the timeout for SetMeta operation for conflict writing.
+| 5000ms
+
+| `cLogPoolGetTimeoutMs`
+| Timeout for getting connection from connection pool
+| 5000ms
+
+| `cLogNetworkRetryCount`
+| Number of attempts for a single doc when there is a network error
+| 5
+
+| `cLogNetworkRetryIntervalMs`
+| Time interval to sleep between attempts to write the conflict doc on network error
+| 2000ms
+
+| `cLogWorkerCount`
+| Number of workers used by the conflict logger, which actually write conflict documents.
+| 20
+
+| `cLogQueueCapacity`
+| Queue capacity for the conflict logger.
+| 6000
+
+| `cLogMaxErrorCount`
+| Max number of errors encountered by conflict logger in a “Error Window” after which the conflict logger is hibernated.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-logger[Conflict Logger].
+| 10
+
+| `cLogErrorTimeWindowMs`
+| This is the time window in which the number of errors in the conflict logger will be checked.
+| 120000ms
+
+| `cLogReattemptDurationMs`
+| This is the duration for which the conflict logger will be hibernated and after this the logger will be active again.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-logger[Conflict Logger].
+| 600000ms
+|===
+
+[#configure-conflictlogging-settings]
+== Enabling and Configuring Conflict Logging
+
+The XDCR Conflict Logging feature can be configured and enabled per replication using the advanced replication setting `conflictLogging`.
+You can use the `conflictLogging` key to mention the storage location for conflict records, and to enable or disable conflict logging.
```

Comment:
Instead of "mention", say "specify" ... 

Instead of "to mention the storage location for conflict records,"
say, "to specify the storage location for conflict records,"